test/serve/serve-basic.test.js

```diff
@@ -7,12 +7,19 @@ const { runWatch } = require('../utils/test-utils');
 jest.setTimeout(20000);
 
 const runServe = args => {
-    return runWatch(path.resolve(__dirname, './shim'), ['serve'].concat(args), false, 'main');
+    return runWatch({
+        testCase: path.resolve(__dirname, './shim'),
+        args: ['serve'].concat(args),
+        setOutput: false,
+        outputKillStr: 'main',
+    });
 };
 
 describe('basic serve usage', () => {
     it('compiles without flags', async () => {
         const { stdout, stderr } = await runServe([]);
+        console.log(stderr);
+        console.log(stdout);
```

Comment:
Why are you logging these to the console?

Reply:
Good catch!